vbuild/velbuild.py

```
@@ -30,8 +35,12 @@ class VELBUILD(APKBUILD):
     @APKBUILD.text.getter
     def text(self) -> str:
         lines: list[str] = []
+        variables = self.variables.copy()
+
+        if self.systemdunits and self.options is None or "!fhs" not in self.options:
```

Ruff (0.15.10)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)